### ISSUE TITLE
Document `jdk.hints.preferences` file format

### ISSUE THREAD

**Issue by kalaracey** [oracle/javavscode#326]
The extension description includes the following section:

> ## Hints Preferences
> Easily update default hint preferences by adjusting a simple configuration option to tailor hint preferences to specific needs.
>
> Go to VSCode `View | Command Palette | Preferences:Open User Settings | Extensions | Java`  and set `Jdk › Hints: Preferences:` option to the xml hint preferences file.

Note that there is no documentation of what this file or its format is. I think what is missing is a link to a Wiki page similar to the link in the Formatting Preferences section to the [Java Formatting Preferences](https://github.com/oracle/javavscode/wiki/Java-formatting-preferences) Wiki page.

See also: https://github.com/oracle/javavscode/issues/239

**Comment by kalaracey** [comment]
I think this is the right place:

<img width="824" alt="Screenshot 2024-11-11 at 4 02 30 PM" src="https://github.com/user-attachments/assets/ae1c7414-944a-4e4f-80bd-61d2a173e7ec">

And then export.

**Comment by Achal1607** [comment]
Hey @kalaracey, thanks for pointing this out. We would add a wiki for hints preferences setting as well.

**Comment by Achal1607** [comment]
Wiki added [Java hint prefs](https://github.com/oracle/javavscode/wiki/Java-Hints-Preferences). Thanks.

**Comment by Britz** [comment]
For me the hints does not work as expected / as configured in Netbeans, e.g., wrong JavaDoc entries are not highlighted.
The formatting works as expected. When clicking the "Export..." button, the same export screen as described in [Java formatting preferences](https://github.com/oracle/javavscode/wiki/Java-formatting-preferences) shows up. Which settings needs to be selected to get it working?

**Comment by nikbobo** [comment]
> For me the hints does not work as expected / as configured in Netbeans, e.g., wrong JavaDoc entries are not highlighted. The formatting works as expected. When clicking the "Export..." button, the same export screen as described in [Java formatting preferences](https://github.com/oracle/javavscode/wiki/Java-formatting-preferences) shows up. Which settings needs to be selected to get it working?

Yes, I also need further configuration instructions to make it work as expected. Clicking export in NetBeans always exports to zip instead of xml. Trying to use a project-level nb-configuration.xml file also didn't work. @Achal1607
